### DISCUSSION TITLE
Outdated release 2.0.1

### DISCUSSION THREAD

**Discussion by jrivard** [JetBrains/xodus#151] [General]
We're long overdue for a release.  Many bugs have been fixed post 2.0.1 but are not released.  I understand a 3.x branch has been in development - but if this isn't ready now we _really_ need a maintenance release of the 2.x branch with the fixes that have long ago been fixed but never released.

**Comment by scrobbleme** [comment]
Any news? I see commits, but no release yet

**Reply by andrii0lomakin** [reply]
@scrobbleme, We have added a statement to the project page that hopefully answers your question.
Will be happy to see you among the users of our new database.
